contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -885,6 +917,243 @@ run_tsql_best_match_heuristics(int nargs, Oid *input_typeids, FuncCandidateList
 	return new_candidates;
 }
 
+/*
+ * getImmediateBaseTypeOfUDT()
+ * This funciton 
+ */
+static Oid getImmediateBaseTypeOfUDT(Oid typeid)
+{
+	Relation					relsetting;
+	HeapTuple					tuple;
+	ScanKeyData					scankey;
+	SysScanDesc					scan;
+	bool						isnull;
+	Datum						datum;
+	Datum                       tsql_typename;
+	LOCAL_FCINFO(fcinfo, 1);
+
+	/* if common_utility_plugin_ptr is not initialised */
+	if (common_utility_plugin_ptr == NULL)
```

Comment:
Redundant, it has already been checked during the initialization.

Reply:
if common_utility_plugin_ptr is not available(i.e. it is NULL) the code following this might run into crash or segmentation fault. So even though this check is redundant, in some unexpected edge case it will be safe to throw error than resulting in crash or segmentation fault.
cc: @Deepesh125

Reply:
yeah, we should always follow this practices.